app/models/ncr/work_order.rb

```diff
@@ -110,6 +110,9 @@ def version
       self.updated_at.to_i
     end
 
+    def name
+      self.project_title
+    end
```

Comment:
Not quite -- `name` is part of the `client_data` interface; it's used in most listing and in a few email templates, iirc.